scout/server/blueprints/cases/static/case_images.js

```diff
@@ -0,0 +1,376 @@
+const WIDTH_BREAKPOINT = 1955  // less will remove one column
```

Comment:
This now works great for two or three columns for me. If you are somewhat certain it will work for all sizes, I'm happy.

There is still/again an issue with less than two columns. The div with the picture goes below the menu etc when this happens. I think we did solve that last time, so we just need to dig out that I guess.

Reply:
At the moment only support for two or three columns. There's a lot of fine-tuning that can be done here -can I add that later?